tests/integration/cairo_path/contracts/starknet_project/basic_starknet_contract_nested_deps.cairo

```diff
@@ -0,0 +1,8 @@
+#[contract]
+mod HelloStarknet {
+    #[external]
+    fn do_something(amount: felt) -> felt {
+        external_lib_bar::bar::bar();
```

Comment:
I assume `bar` is lib with nested dependency. Can we add a test for tests compilation with nested dependency as well?

Reply:
[add nested deps to test](https://github.com/software-mansion/protostar/pull/1433/commits/f48d2087e666bcdd407040b1e7624d76b5df747a)